bouffalo-hal/src/sec/aes.rs

```diff
@@ -438,19 +439,13 @@ impl Endianness {
         self.0 |= (len << 30) & Self::COUNTER_LENGTH;
     }
 
-    /// 获取CTR模式下计数器的长度
     /// Get counter length for CTR mode
     #[inline]
     pub fn get_counter_length(&self) -> u32 {
         (self.0 & Self::COUNTER_LENGTH) >> 30
     }
```

Comment:
```suggestion
    pub fn counter_length(&self) -> u32 {
        (self.0 & Self::COUNTER_LENGTH) >> 30
    }
```

Rust-style getters usually does not include the `get_` prefix.